cli/foundation-models/system/inference/image-to-text/image-to-text-batch-endpoint.sh

```diff
@@ -0,0 +1,111 @@
+set -x
+# The commands in this file map to steps in this notebook: https://aka.ms/azureml-infer-batch-sdk-image-classification
```

Comment:
Nit: Should the link be updated to corresponding image-to-text notebook

Reply:
fixed